README.md

```diff
@@ -98,7 +98,7 @@ $ helm install addons pega/addons --namespace pegaaddons --values addons.yaml
 $ helm install mypega pega/pega --namespace mypega --values pega.yaml
 ```
 
-*To edit the charts and deploy using a local version of the pega/addons, pega/backingservices, or pega/pega charts, you must include the filepath to your local chart directory in your Helm chart reference.*
+To edit the charts and deploy using a local version of the pega/addons, pega/backingservices, or pega/pega charts, you must include the filepath to your local chart directory in your Helm chart reference.
```

Comment:
I approve of all of your changes. Fantastic! For this one, you may have a conflict. I saw a new merge happened and updated this section appropriately:
Important: To edit the charts and deploy using a local version of the pega/addons, pega/backingservices, or pega/pega charts, you must include the filepath to your local chart directory in your Helm chart reference.

Tip: To delete your deployment of Pega nodes, enter the command (this will not delete your database):

$ helm delete release --namespace mypega